src/fragments/guides/api-graphql/common/subscriptions-by-id.mdx

```diff
@@ -0,0 +1,58 @@
+Take for example the following GraphQL schema:
+
+```graphql
+type Post @model @auth(rules: [{allow: public, provider: apiKey}]){
```

Comment:
uber nit: Could we please put a space between the brackets and rule defs like on https://docs.amplify.aws/cli/graphql/authorization-rules/ for consistency? e.g. `{ allow: public, provider: apiKey }` for all the auth rules added here?